docs/source/contributing.rst

```diff
@@ -157,6 +157,21 @@ To update these links use the following commands, after setting the following en
     travis env set --repo mesosphere/dcos-e2e EE_1_10_ARTIFACT_URL $EE_1_10_ARTIFACT_URL
     travis env set --repo mesosphere/dcos-e2e EE_1_11_ARTIFACT_URL $EE_1_11_ARTIFACT_URL
 
+Updating the Amazon Web Services credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Private credentials for Amazon Web Services used by Travis CI.
+
+To update the credentials use the following commands, after setting the following environment variables:
+
+* ``AWS_ACCESS_KEY_ID``
+* ``AWS_SECRET_ACCESS_KEY``
```

Comment:
Note where in OneLogin the currently used keys are from.

Reply:
Noted below.